README.md

```diff
@@ -1,2 +1,131 @@
-# technical-glossary
-A technical glossary for key words
+# Technical Glossary
+
+### Why?
+Have you ever learnt something cool in coding without knowing what it's actually
+called? Or have you ever been reading about a concept only to get lost in the
+number of acronyms and other technical words used to describe what you're
+learning about?
+
+This resource has been created to provide quick access, simple and easy to
+understand definitions to stop big words getting in the way of what could be
+happy learning :blush: :muscle:
+
+### What?
+
+A technical glossary for key words. These definitions are by no means exhaustive
```

Comment:
"_by no means exhaustive_" could be changed to "_deliberately_ succinct". 
We should link to the full wikipedia article or additional reading wherever appropriate 
for people to extend their knowledge if they feel the need.